src/test/unit_tests/test_nb_native_fs.js

```diff
@@ -0,0 +1,173 @@
+/* Copyright (C) 2016 NooBaa */
+'use strict';
+
+// const _ = require('lodash');
+const fs = require('fs');
+const mocha = require('mocha');
+const assert = require('assert');
+const nb_native = require('../../util/nb_native');
+
+mocha.describe('nb_native fs', function() {
+
+    // mocha.describe('stat', function() {
```

Comment:
what's up with this test? Why is it all commented out?

Reply:
I will fill this up in the next PR as well